test/infrastructure/docker/exp/api/v1beta1/dockermachinepool_webhook.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+func (r *DockerMachinePool) SetupWebhookWithManager(mgr ctrl.Manager) error {
```

Comment:
In theory that's not necessary as the other webhooks already registery the conversion webhook, but that would be too implicit.